### PR TITLE
github_config: add an action to validate markdown links

### PR DESCRIPTION
**Pull request by prince-chrismc**
There are a handful of broken links in the docs. It would be nice to catch them in PRs and double check master.

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/workflows/marldown-links.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Check Markdown links
+
+on: [push,  pull_request]
```
Comment (jgsogo) [inline .github/workflows/marldown-links.yml:3]: I would use only `pull_request`, we don't want to see a _red_ when things are merged to `master`

File: .github/workflows/mlc_config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "ignorePatterns": [
+      {
+        "pattern": "^https://github.com/"
+      },
+      {
+        "pattern": "^https://docs.github.com/"
+      }
+    ]
+}
```
Comment (jgsogo) [inline .github/workflows/mlc_config.json:10]: Why this `ignorePatterns`? Maybe better to fix the issue this way https://github.com/gaurav-nelson/github-action-markdown-link-check#github-links-failure-fix

File: .github/workflows/marldown-links.yml (hunk 1 above)
Comment (prince-chrismc) [reply .github/workflows/marldown-links.yml:3]: My goal was allow it to run when we work in forks branches.

I'll make sure it wont be red here

File: .github/workflows/mlc_config.json (hunk 2 above)
Comment (prince-chrismc) [reply .github/workflows/mlc_config.json:10]: I do not remember why I did it this way 🤔

Comment (prince-chrismc) [reply .github/workflows/mlc_config.json:13]: It was github rate limiter

but it's working this time 🤷

**Review by uilianries — APPROVED** [review]

**Pull request merged**